### ISSUE TITLE
Confusing question/answer on Metaprogramming Quiz

### ISSUE THREAD

**Issue by LNKeat** [learn-co-curriculum/phase-3-quiz-mass-assignment#1]
### Canvas Link

https://learning.flatironschool.com/courses/4323/quizzes/19730

### Concern

The correct answer on this quiz does not seem to line up with the way the material leading up to this question is worded.  
Screenshots & quotes are below.  
<img width="634" alt="Screen Shot 2021-12-23 at 12 06 08 PM" src="https://user-images.githubusercontent.com/81266679/147284445-b1e64f46-968d-4305-8242-15b301b994f3.png">

### Additional Context

Reference Quotes: 

 ###Introduction to Metaprogramming:
> In this section we'll be discussing and working through the **following metaprogramming concepts**:
> Mass assignment
> _*Keyword arguments*_

 ### Mass Assignment and Metaprogramming:
> Guess what? We can achieve exactly that goal using metaprogramming **and** mass assignment. Let's take a look at how it's done, then we'll break it down together.

### QUESTION
Consider the following code:
```
class User
  attr_accessor :name, :age, :location, :user_name

  def initialize(user_name:, name:, age:, location:)
    @user_name = user_name
    @name = name
    @location = location
    @age = age
  end
end
```
### ANSWER
The above code snipet is considered metaprogramming.  (True or False)  
The correct answer is False.  

The two quotes I put above imply that the use of  "keyword arguments" is at least part of metaprogramming.  
The only topic preceding the Mass Assignment Lesson is the Keyword Arguments Section and the lesson used to reference the "correct" answer to this question references **Metaprogramming** AND Mass Assignment
Does this not imply that keyword arguments ARE metaprogramming?  
<img width="913" alt="Screen Shot 2021-12-23 at 12 05 45 PM" src="https://user-images.githubusercontent.com/81266679/147284364-64ca6df9-4d67-42bc-bc62-44c210c59c2f.png">
<img width="904" alt="Screen Shot 2021-12-23 at 12 06 37 PM" src="https://user-images.githubusercontent.com/81266679/147284370-1ca305b4-844a-4fe8-9397-cb30e08cad45.png">

### Suggested Changes

Though the answer to this question may technically be "false"...  this question at the very least is a "trick" question and definitely is not clearly explained in the lessons that lead up to this.  

I would suggest rewording the question so that it's not meant to trick us 
OR
change the curriculum wording in the multiple lessons preceding the quiz to make it clear whether keyword arguments ARE metaprogramming or only an ingredient.

**Comment by ihollander** [comment]
Hi @LNKeat - thanks for your feedback! I can see how this would be confusing based on the fact that the metaprogramming lesson lists keyword arguments as a concept related to metaprogramming. We'll update this question in the quiz so the initialize method in the example doesn't use keyword arguments to make it clearer that false is the correct answer.
